homeassistant/components/zha/core/device.py

```diff
@@ -557,7 +562,15 @@ async def issue_cluster_command(
 
     async def async_add_to_group(self, group_id):
         """Add this device to the provided zigbee group."""
-        await self._zigpy_device.add_to_group(group_id)
+        try:
+            await self._zigpy_device.add_to_group(group_id)
```

Comment:
should we behave differently in case of a failure?  Or do we act on `zigpy` events for successful additions?

Reply:
We react on the events